crates/astria-sequencer-types/src/sequencer_block_data.rs

```diff
@@ -95,15 +118,16 @@ impl SequencerBlockData {
         &self.rollup_data
     }
 
-    #[allow(clippy::type_complexity)]
     #[must_use]
-    pub fn into_values(self) -> (Hash, Header, Option<Commit>, HashMap<Namespace, RollupData>) {
-        (
-            self.block_hash,
-            self.header,
-            self.last_commit,
-            self.rollup_data,
-        )
+    pub fn into_raw(self) -> RawSequencerBlockData {
```

Comment:
Please add a comment on top of this function.

Nit: destructuring `self ` looks slightly nicer (less `self.`)
```rust
    let Self {
        block_hash,
        header,
        // and the rest
    } = self;
    RawSequencerBlockData {
        block_hash,
        header,
        // and the rest
    }
```